src/pages/share/safe-app.tsx

```diff
@@ -28,9 +29,15 @@ const ShareSafeApp = () => {
   }
 
   return (
-    <main>
-      <SafeAppLanding appUrl={appUrl} chain={chain} />
-    </main>
+    <>
+      <Head>
+        <title>Safe Apps – Share</title>
```

Comment:
Perhaps we can include the app name in the title as a future enhancement 🤔

Reply:
yep, we sent the app name or url (if custom) [here](https://github.com/safe-global/web-core/blob/39776b65ac58cc0b71a720fcabd62a66eb3f883e/src/components/safe-apps/SafeAppLandingPage/index.tsx#L36)